### ISSUE TITLE
Improve accessibility by adding title attribute to iframes

### ISSUE THREAD

**Issue by shannoding** [CMU-313/Teedy#313]
Teedy uses iframes to display any document uploads. The Accessibility score on the document display page is 83 as the <iframe> is missing a title attribute.

**Comment by shannoding** [comment]
I would like to work on this please!
